diff --git a/node_modules/phaser/dist/phaser.esm.js b/node_modules/phaser/dist/phaser.esm.js
index 32e7581..2755f82 100644
--- a/node_modules/phaser/dist/phaser.esm.js
+++ b/node_modules/phaser/dist/phaser.esm.js
@@ -176843,6 +176843,9 @@ var RenderTarget = new Class({
     {
         var renderer = this.renderer;
 
+        width = Math.max(Math.round(width * this.scale), 1);
+        height = Math.max(Math.round(height * this.scale), 1);
+
         this.texture = renderer.createTextureFromSource(null, width, height, this.minFilter, this.forceClamp);
         this.framebuffer = renderer.createFramebuffer(width, height, this.texture, this.hasDepthBuffer);
 
@@ -176902,6 +176905,9 @@ var RenderTarget = new Class({
         {
             var renderer = this.renderer;
 
+            width = Math.max(Math.round(width * this.scale), 1);
+            height = Math.max(Math.round(height * this.scale), 1);
+
             renderer.deleteFramebuffer(this.framebuffer);
 
             renderer.deleteTexture(this.texture);
diff --git a/node_modules/phaser/dist/phaser.js b/node_modules/phaser/dist/phaser.js
index 51b1458..b5d354d 100644
--- a/node_modules/phaser/dist/phaser.js
+++ b/node_modules/phaser/dist/phaser.js
@@ -176951,6 +176951,9 @@ var RenderTarget = new Class({
     {
         var renderer = this.renderer;
 
+        width = Math.max(Math.round(width * this.scale), 1);
+        height = Math.max(Math.round(height * this.scale), 1);
+
         this.texture = renderer.createTextureFromSource(null, width, height, this.minFilter, this.forceClamp);
         this.framebuffer = renderer.createFramebuffer(width, height, this.texture, this.hasDepthBuffer);
 
@@ -177010,6 +177013,9 @@ var RenderTarget = new Class({
         {
             var renderer = this.renderer;
 
+            width = Math.max(Math.round(width * this.scale), 1);
+            height = Math.max(Math.round(height * this.scale), 1);
+
             renderer.deleteFramebuffer(this.framebuffer);
 
             renderer.deleteTexture(this.texture);
